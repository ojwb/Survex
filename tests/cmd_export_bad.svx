; pos=fail warn=1 error=2
; We used to report "End of line not blank" but 1.2.22 gives "Character "=" not
; allowed in station name (use *SET NAMES to set allowed characters)" instead.
*fix x.1 0 0 0
*begin x
*export *
*export 1 =
*export 2 
*export 3 ; test
*export 4
1 2 1.00 010 01
2 3 1.00 010 01
*end x
