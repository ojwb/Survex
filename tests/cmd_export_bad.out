./cmd_export_bad.svx:6:9: error: Character "*" not allowed in station name (use *SET NAMES to set allowed characters)
 *export *
         ^
./cmd_export_bad.svx:7:11: error: Character "=" not allowed in station name (use *SET NAMES to set allowed characters)
 *export 1 =
           ^

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 3 survey stations, joined by 2 legs.
There are 0 loops.
Total length of survey legs =    2.00m (   2.00m adjusted)
Total plan length of survey legs =    2.00m
Total vertical length of survey legs =    0.03m
Vertical range = 0.03m (from x.3 at 0.03m to x.1 at 0.00m)
North-South range = 1.97m (from x.3 at 1.97m to x.1 at 0.00m)
East-West range = 0.35m (from x.3 at 0.35m to x.1 at 0.00m)
./cmd_export_bad.svx:10: warning: Station "x.4" referred to by *entrance or *export but never used
   2 1-nodes.
   1 2-node.
There were 1 warning(s) and 2 error(s) - no output files produced.
